instructions/section-2/index.html

```diff
@@ -49,15 +49,22 @@ <h4>P5 JS Javascript Project Pre-Assignment</h4>
         <a href="https://editor.p5js.org/EdithHarrison/full/76ZVQtvIV" style="text-decoration: none;">https://editor.p5js.org/EdithHarrison/full/76ZVQtvIV</a>
         <p>This project is created using P5.js and features a spider that moves downward and picks up a ball.</p>
     </section>
-
+    
+    <!-- Add Skills Section -->
+    <section id="skills" style="padding: 20px; margin: 20px; border-radius: 5px;">
```

Comment:
Note: It's best practice to add css to a .css file and not have it directly on the html element like you do here.